pkg/controller/control.go

```diff
@@ -584,6 +584,14 @@ func (c *Controller) Start(addresses ...string) error {
 		return nil
 	}
 
+	checkDuplicate := map[string]struct{}{}
+	for _, address := range addresses {
+		if _, exist := checkDuplicate[address]; exist {
+			return fmt.Errorf("invalid ReplicaAddress: duplicate replica addresses")
```

Comment:
Could you have a format string w/ the duplicated address inside? `("invalid ReplicaAddress: duplicate replica addresses %s", address)`